requirements/test-requirements.txt

```diff
@@ -17,9 +17,7 @@ kaleido
 # Required by tuning tests
 hyperopt
 # Required by recipes tests
-# Pinned because ipython>=8.13 doesn't support Python 3.8,
-# which is MLflow's minimum supported Python version
-ipython<8.13.0
+ipython
```

Comment:
@serena-ruan pip should automatically pick up a version that's installed in the current python version.

Reply:
Yes true. If it's not installed it should install the latest available version?

Reply:
yep!